src/core/mavlink_parameters.h

```diff
@@ -238,25 +222,25 @@ class MAVLinkParameters {
 
         bool set_as_same_type(const std::string& value_str)
         {
-            if (_value.is<uint8_t>()) {
+            if (std::get_if<uint8_t>(&_value)) {
```

Comment:
I suppose `_value.is` does not work with a variant, hence this change?

Reply:
I think so. There is also `std::get` which throws instead: https://en.cppreference.com/w/cpp/utility/variant/get